src/Config.php

```diff
@@ -134,11 +134,11 @@ public function define(Container $container)
             && is_array($dependencies['invokables'])
         ) {
             foreach ($dependencies['invokables'] as $service => $class) {
-                if ($service !== $class) {
+                if (! is_int($service) && $service !== $class) {
```

Comment:
Would probably prefer `is_string` instead but at the end same result.

Reply:
Good point, I'll change it!